client/service/src/config.rs

```diff
@@ -191,16 +184,13 @@ impl<C, G, E> Configuration<C, G, E> where
 			tracing_targets: Default::default(),
 			tracing_receiver: Default::default(),
 		};
-		configuration.network.boot_nodes = configuration.chain_spec.boot_nodes().to_vec();
```

Comment:
Where is this done now?

Reply:
https://github.com/paritytech/substrate/pull/4692/files#diff-ed4918dd0da951e344e205c78f972484R312